spec/spec.md

```diff
@@ -192,6 +192,7 @@ sequenceDiagram
 - For transportation of VCs, First Implementer's Draft of OpenID for Verifiable Presentations MUST be used as defined in [[ref: OpenID4VP ID1]].
 - As the query language, [[ref: Presentation Exchange v1.0.0]] MUST be used and conform to the syntax defined in [[ref: OpenID4VP ID1]].
 - Decentralized Identifiers (DIDs), as defined in [[ref: DID Core]], MUST be used as identifiers of the entities. Implementations MUST support did:web and did:ion as a mandatory DID method as defined in [[ref: did-web]] and [[ref: did-ion]].
+- DID Documents MUST use `publicKeyJwk` for their Verification Material ([[ref: DID Core]] section 5.2.1).  Keys MUST be secp256k1 or Ed25519, see the _Cryptographic Signature_ section.
```

Comment:
Suggest that the DID document may contain verification material that is used beyond the scope of this profile. As an example, the document might contain a verification key that `publicKeyJwk` and other keys that might not be.
i.e., the language should be about the verification material actually being used for this interop profile rather than the document as a whole.

Reply:
Is the intent of this PR to profile the verification method type? (e.g., `JSON Web Signature 2020`)? And then, if necessary, profile each of those verification method types further (e.g., which key encoding property are allowed)?

Reply:
And presumably `JSON Web Signature 2020`, itself, does not need additional profiling for this topic since it is already `publicKeyJwk`.